index.js

```diff
@@ -24,6 +24,12 @@ program
       }
     }
 
+    var num = numberOfResults;
+
+    if(numberOfResults === undefined) {
```

Comment:
You can't use `=== undefined` when trying to figure out if a variable is undefined, you need to use
`if (typeof numberOfResults === 'undefined')`